src/plays/dad-jokes/Readme.md

```diff
@@ -0,0 +1,52 @@
+# Dad Jokes 🔥
+
+### Dad Jokes is a fully functional react app which offers the various categories of jokes to enjoy.
+
+## Features
+
+- ### **This app is fully responsive across all the devices 📱**
+- ### **This app has Dark Mode & Light Mode 💯**
+- ### **User can enjoy different categories of jokes like Coding, Miscellaneous, Puns & Random 🥳**
+- ### **User can upvote and downvote any joke 🚀**
+- ### **Each joke has a Emoticon, changes on the basis of votes 🪄**
+- ### **Animations - Upvotes, Downvotes & Emojis have smooth animations ⚡**
+
+## Play Screenshot
+
+- ### **Light Mode** 
+
+<div align="center">
+  <img src="https://i.postimg.cc/0N2n74Bf/light-mode.png" alt="Dad Jokes Screenshot" width="100%" target="_blank" />
+  <br>
+</div>
+
+- ### **Dark Mode** 
+<div align="center">
+  <img src="https://i.postimg.cc/3JXFV7M1/dark-mode.png" alt="Dad Jokes Screenshot" width="100%" target="_blank" />
+  <br>
+</div>
+
+## Play Demographic
+
+- ### **Language: JavaScript**
+- ### **Level: Intermediate**
+
+## Creator Information
+
+- ### **Bobby Sadhwani** - **[Linkedin](https://www.linkedin.com/in/bobby-sadhwani)**, **[Twitter](https://twitter.com/BOBBY__SADHWANI)**, **[Github](https://github.com/Bobby-coder)**, **[Portfolio](https://bobby-sadhwani.netlify.app)**  
+
+## Video
+
+- ### **[stackstream](https://www.stack-stream.com/v/dad-jokes-app-12)**
```

Comment:
Please edit the play and add this stream link in the video field. Make sure the link is publicly accessible.

Reply:
Done